### ISSUE TITLE
Dialog opened via service is missing wrapper attributes

### ISSUE THREAD

**Issue by Raisolution** [telerik/kendo-angular#3523]
When the Dialog is opened via service it is missing the `tabindex` and `dir` wrapper attributes. This leads to issues with accessibility and keyboard navigation.

Example to reproduce:
https://www.telerik.com/kendo-angular-ui/components/dialogs/dialog/service/
